### PR TITLE
single-file module artifact generation command implementation

### PR DESCRIPTION
**Pull request by mchalski**
hopefully the last one of:

https://tracker.mender.io/browse/MEN-3039

@oleorhagen FYI for your pr.

### CODE REVIEW

File: log/log.go [hunk 1]
```diff
@@ -24,15 +24,15 @@ func Init(verbose bool) {
 }
 
 func Error(fmt string, args ...string) {
-	log.Printf("[ERROR] "+fmt, args)
+	log.Printf("ERROR \n"+fmt, args)
```
Comment (tranchitella) [inline log/log.go]: Are you sure about the newline position?

File: log/log.go [hunk 2]
```diff
@@ -24,15 +24,15 @@ func Init(verbose bool) {
...
 
 func Info(fmt string, args ...string) {
-	log.Printf("[INFO] "+fmt, args)
+	log.Printf(fmt+"\n", args)
 }
 
 func Verbose(fmt string, args ...string) {
 	if isVerbose {
-		log.Printf("[VERBOSE] "+fmt, args)
+		log.Printf(" -- \n"+fmt, args)
```
Comment (tranchitella) [inline log/log.go]: same here

File: log/log.go (hunk 1 above)
Comment (mchalski) [reply log/log.go]: yup, fixed.

File: log/log.go (hunk 2 above)
Comment (mchalski) [reply log/log.go]: fixed as well

**Pull request merged**